package.json

```diff
@@ -104,6 +104,7 @@
 		"@hashicorp/platform-markdown-utils": "^0.2.2",
 		"@hashicorp/platform-packer-plugins": "^0.3.0",
 		"@hashicorp/platform-product-meta": "^0.1.1",
+		"@hashicorp/platform-util": "^0.2.0",
```

Comment:
I'm using `mitigateWidows` to try to get the `description` text wrapping nicely, so relying directly on `@hashicorp/platform-util`.